lib/view/templates.py

```diff
@@ -361,6 +346,51 @@ def usage_weighted_avg(edatas: list[EntryData]):
                 ),
             ),
         ),
+        Subgroup(
+            "Secondary Index",
+            (
+                Field("Type", Projectors.String("ns_stats", "sindex-type")),
+                Field(
+                    "Used",
+                    Projectors.Number(
+                        "ns_stats",
+                        "sindex_flash_used_bytes",
+                        "sindex_pmem_used_bytes",
+                        "memory_used_sindex_bytes",
+                    ),
+                    converter=Converters.byte,
+                    aggregator=Aggregators.sum(),
+                ),
+                Field(
+                    "Used%",
+                    Projectors.PercentCompute(
+                        Projectors.Number(
+                            "ns_stats",
+                            "sindex_flash_used_bytes",
+                            "sindex_pmem_used_bytes",
+                            "memory_used_sindex_bytes",
+                        ),
+                        Projectors.Number("ns_stats", "sindex-type.mounts-size-limit"),
+                    ),
+                    converter=Converters.round(2),
+                    aggregator=ComplexAggregator(
+                        create_usage_weighted_avg("Secondary Index"),
+                        converter=Converters.round(2),
```

Comment:
I'm guessing this will need to change to use what is in https://github.com/aerospike/aerospike-admin/pull/121 ?

Reply:
Correct